packages/firebase_ml_vision/lib/src/cloud_detector_options.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This is just a file rename `vision_cloud_detector_options.dart` -> `vision_cloud_detector_option.dart`